### ISSUE TITLE
Unable to use with react-helmet

### ISSUE THREAD

**Issue by cia48621793** [zeit/next.js#1246]
The node was not even rendered...

**Comment by albinekb** [comment]
Have you looked at the docs?
https://github.com/nfl/react-helmet#server-usage

Is this using next@beta?

**Comment by cia48621793** [comment]
@albinekb where should I put my Helmet.rewind?

**Comment by cia48621793** [comment]
@albinekb latest

**Comment by cia48621793** [comment]
@albinekb 
I have the _document.js set up like this tho:
```
import Document, { Head, Main, NextScript } from 'next/document'
import Helmet from 'react-helmet'

export default class MyDocument extends Document {
  render () {
    return (
     <html>
       <Head>
         <Helmet
           link={[
             {rel:'stylesheet', href: "https://maxcdn.bootstrapcdn.com/bootstrap/latest/css/bootstrap.min.css"},
             {rel:'stylesheet', href: "https://cdnjs.cloudflare.com/ajax/libs/bootstrap-material-design/4.0.2/bootstrap-material-design.min.css"}
           ]}
         />
       </Head>
       <body>
         <Main />
         <NextScript />
       </body>
     </html>
    )
  }
}
```
I couldn't however find anywhere to put Helmet.rewind, where's the post render function?

**Comment by impronunciable** [comment]
@cia48621793 this is a valid question, although we recommend using `next/head` for dealing with the page `<head>` https://github.com/zeit/next.js#populating-head

**Comment by impronunciable** [comment]
@cia48621793 About the `Helmet.rewind` you do it inside the `getInitialProps` document class method like in this example https://github.com/zeit/next.js/blob/master/examples/with-styled-components/pages/_document.js#L7, then you can pass the `head` as a initial prop to the document to inject into the page.

**Comment by cia48621793** [comment]
[Closed as of #1264](https://github.com/zeit/next.js/pull/1264)
